q2_longitudinal/_utilities.py

```diff
@@ -814,6 +814,7 @@ def _summarize_feature_stats(table, state_md_col):
 # df.fillna(None) does not work so used solution from:
 # https://github.com/pandas-dev/pandas/issues/1972
 def _convert_nan_to_none(df):
+    df = df.astype(object)
```

Comment:
I just fixed another issue caused by this same kinda thing [here](https://github.com/qiime2/q2-types/pull/294). Annoying af